src/Copy/Text.elm

```diff
@@ -11,7 +11,7 @@ t : Key -> String
 t key =
     case key of
         SiteTitle ->
-            "[cCc] SEA MAP"
+            "Economic abuse support tool"
```

Comment:
I would title case this.. but one for when we have a real title.

Reply:
There will be a [cCc] in the index.html file as well. I think we found before that it flickers as the Elm app is loaded - also not too important to fix now.

Reply:
Good catch. Done.